src/lib/CrsfProtocol/crsf_protocol.h

```diff
@@ -107,6 +107,7 @@ typedef enum
     CRSF_FRAMETYPE_MSP_WRITE = 0x7C, // write with 8 byte chunked binary (OpenTX outbound telemetry buffer limit)
     // Ardupilot frames
     CRSF_FRAMETYPE_ARDUPILOT_RESP = 0x80,
+    CRSF_FRAMETYPE_ARDUPILOT_LUA = 0x81, // bidirectional telemetry via standard (not extended) CRSF frame
```

Comment:
This addition might hold up the PR.  Im all for expanding the CRSF protocol but we need to find a way to do it in a controlled manner with TBS.  For now it might be best to remove it from the PR while we figure out how best to do this.

Reply:
I agree. Please go ahead and remove this addition. I"m still working on the PR with Ardupilot on this topic, and will open a new PR here if needed.

Is there any official CRSF documentation available? Would help me a lot.